us.zoom.Zoom.json

```diff
@@ -12,10 +12,13 @@
         "--share=ipc",
         "--socket=x11",
         "--socket=pulseaudio",
+        "--socket=session-bus",
         "--share=network",
         "--device=all",
+        "--filesystem=xdg-run/pipewire-0",
         "--filesystem=~/Documents/Zoom:create",
         "--filesystem=~/.zoom:create",
+        "--filesystem=xdg-run/pipewire-0",
```

Comment:
This line appears to have already been added [above](https://github.com/flathub/us.zoom.Zoom/pull/284/files#diff-29c6733da0fa2e3285ba00b2364abc5f54b65feb2f7209d5dfc4351fcd2d9496R18)